app/liquid_tags/glitch_tag.rb

```diff
@@ -36,6 +37,7 @@ def valid_id?(input)
 
   def parse_id(input)
     id = input.split(" ").first
+    id.gsub!(TILDE_PREFIX_REGEXP, "")
```

Comment:
no need to `gsub!` if the regexp uses `\A` since it's just going to scan the beginning of the string, `sub!` is fine: 

```ruby
[2] pry(main)> "~some-id".gsub!(/\A~/, "")
=> "some-id"
[3] pry(main)> "~some-id".sub!(/\A~/, "")
=> "some-id"
```